components/landing/highlights.json

```diff
@@ -89,7 +89,7 @@
       "title": "Pervasive media censorship",
       "text": "Blocking of hundreds of media websites",
       "report": "https://ooni.io/post/egypt-internet-censorship/",
-      "explore": "/search?input=madamasr.com&probe_cc=EG",
+      "explore": "/search?domain=madamasr.com&probe_cc=EG",
```

Comment:
We should add some date ranges to this query otherwise the results will change depending on when somebody loads this page. cc @agrabeli

Reply:
Added.